containers/daap-landing-to-raw/src/var/task/landing_to_raw.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import boto3
+from data_platform_logging import DataPlatformLogger, s3_security_opts
+from data_platform_paths import RawDataExtraction, get_raw_data_bucket
+
+s3 = boto3.client("s3")
+
+
+def handler(event, context):
+    raw_data_bucket = get_raw_data_bucket()
+    bucket_name = event["detail"]["bucket"]["name"]
+    file_key = event["detail"]["object"]["key"]
+    copy_source = {"Bucket": bucket_name, "Key": file_key}
+    destination_key = file_key.replace("landing/", "raw/")
+
+    config = RawDataExtraction.parse_from_uri("s3://bucket/" + file_key)
+    logger = DataPlatformLogger(
+        extra={
+            "image_version": os.getenv("VERSION", "unknown"),
+            "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+            "data_product": config.element.data_product.name,
+            "table": config.element.curated_data_table.name,
```

Comment:
Could these be consistently named? like https://github.com/ministryofjustice/data-platform/blob/main/containers/daap-athena-load/src/var/task/athena_load_handler.py#L33-L34
We should probably think of a better way to do this. Maybe within the base image logging module when passed a `RawDaraExtraction` object, but think that's not one for this PR